### ISSUE TITLE
Deployment on Kubernetes

### ISSUE THREAD

**Issue by harshmaur** [flutterjanus/JanusDocker#1]
Hey @shivanshtalwar0, are you using kubernetes to deploy this?

**Comment by shivanshtalwar0** [comment]
As of now i deployed it with just docker run command on VM or bare metal servers
But deployment on k8s is on my todo list

**Comment by harshmaur** [comment]
Do you have an idea how networking would work in kubernetes?

**Comment by shivanshtalwar0** [comment]
Although i am still exploring kubernetes, but considering what i know about kubernetes and janus we will need to create a networking service which would make load balancer to proxy all incoming request to our janus container running in pods we will need to expose networking ports such as 8088(rest) 8188(websocket) and port range 10000-50000 for other essential purposes like rtp forward and all 
In short networking stack is the key to setup  janus on k8s i will update this issue once i try it in real.

**Comment by getsolutionstz** [comment]
> Although i am still exploring kubernetes, but considering what i know about kubernetes and janus we will need to create a networking service which would make load balancer to proxy all incoming request to our janus container running in pods we will need to expose networking ports such as 8088(rest) 8188(websocket) and port range 10000-50000 for other essential purposes like rtp forward and all In short networking stack is the key to setup janus on k8s i will update this issue once i try it in real.

Hello, @shivanshtalwar0 any update on this. i have tried running Janus on kubernetes for weeks now not succesfull.

**Comment by shivanshtalwar0** [comment]
@getsolutionstz Unfortunately, it is not really possible to deploy Janus on kubernetes because of large port requirements of janus and state full nature of webrtc

**Comment by getsolutionstz** [comment]
How can I implement a scalable Janus cluster where users are connected to different Janus instances but are part of the same room in my application? Specifically, I am considering two approaches:

Distributed Rooms: UserA connects to Janus1 and UserB to Janus2, with each user in separate rooms in Janus but sharing the same room in my application. In this scenario, each user would play streams from the other Janus instance.

RTP Forwarding: Is it feasible to create a room across all Janus instances that receive RTP-forwarded streams from a single Janus instance?

Which approach would be MORE EFFECTIVE, or is there an alternative method for achieving this kind of distributed but unified room experience in a Janus WebRTC setup?

**Comment by shivanshtalwar0** [comment]
This is the core idea of my startup so i can't share the exact strategy to you sorry, however you are welcome to visit my product website https://januscaler.com soon the apis will be available for public consumption at an affordable pricing
